### ISSUE TITLE
Nullness Checker Static Initialization

### ISSUE THREAD

**Issue by GoogleCodeExporter** [typetools/checker-framework#353]
```
What steps will reproduce the problem?

The Nullness Checker fails to warn of a NullPointerException for this code:

class StaticDataflow {

    static String a;

    static {
        a.toString();
    }
}

What is the expected output? What do you see instead?
I expect a warning on the dereference of "a", but no error is given.

What version of the product are you using? On what operating system?
261dee828a0a

```

Original issue reported on code.google.com by `mcart...@cs.washington.edu` on 4 Sep 2014 at 9:35
